libraries/adaptive-expressions/src/expressionFunctionsForNode.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ExpressionEvaluator } from './expressionEvaluator';
+import { ExpressionType } from './expressionType';
+import { ExpressionFunctions } from './expressionFunctions';
+import { ReturnType, Expression } from './expression';
+import * as os from 'os';
+
+/**
+ * Inject Node-specific functions into the common builtin-function list.
+ */
+export class ExpressionFunctionsForNode {
+    public static _initialize(): void {
```

Comment:
If you do want this public, then I would avoid the _ prefix.
If it is public, what happens if someone calls it again?

Reply:
That's a good point. I will fix this one.